integration/src/test/java/org/mule/test/integration/routing/outbound/UntilSuccessfulRetryExhaustedTestCase.java

```diff
@@ -34,13 +34,20 @@ protected String getConfigFile() {
   }
 
   @Test
-  public void onRetryExhaustedCallExceptionStrategy() throws Exception {
+  public void onRetryExhaustedCallExceptionStrategyWithRetryExhaustedError() throws Exception {
```

Comment:
better add a new test case for this scenario, this one is already testing something else.

Reply:
Agree. I will make the changes :)